### PR TITLE
Crazy4pi314/updating readme

### PR DESCRIPTION
**Pull request by crazy4pi314**
Improving readme and adding some missing files for a contribution guide and code of conduct.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at sckaiser@sckaier.com. All
```
Comment (glassnotes) [inline CODE_OF_CONDUCT.md:58]: There's a typo in your e-mail address!

File: README.md [hunk 2]
```diff
@@ -1,61 +1,98 @@
...
+Check out the [primer on memory for quantum computers]() in our docs!
 
-## General project goals
+## Motivation
 
-- Proficiency with Q# as a programming language
-- Understanding the role of qRAM in quantum computing, its benefits, and costs
-- Describe the different memory paradigms in quantum machine learning
+There are many different proposals for qRAM in quantum computing that each have different tradeoffs, and currently come up a lot in quantum machine learning applications.
+We want to better understanding the costs and benefits of different qRAM implementations in quantum machine learning as well as quantum computing more generally.
```
Comment (glassnotes) [inline README.md:10]: *understand

File: README.md [hunk 3]
```diff
@@ -1,61 +1,98 @@
...
+- [Quantum Development Kit](https://docs.microsoft.com/quantum/)
+- [.NET Core SDK 3.1](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- [Python](https://www.python.org/downloads/)
+- [Visual Studio Code](https://code.visualstudio.com/) and [Visual Studio](https://visualstudio.microsoft.com/)
+- [Jupyter Notebook](https://jupyter.org/)
 
-### Step 5
-- Writing up findings, documenting code, and getting ready for release
-    - Publish on Blog etc.
+## Features
```
Comment (glassnotes) [inline README.md:42]: A lot of this stuff isn't filled out, is this just part of the template and we will complete it later?

**Review by glassnotes — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (crazy4pi314) [reply README.md:42]: Yeah thats the idea!

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (crazy4pi314) [reply CODE_OF_CONDUCT.md:58]: 🤦

**Pull request merged**